include/bluetooth/mesh/sensor.h

```diff
@@ -1160,6 +1160,116 @@ const struct bt_mesh_sensor_type *bt_mesh_sensor_type_get(uint16_t id);
 const struct bt_mesh_sensor_format *
 bt_mesh_sensor_column_format_get(const struct bt_mesh_sensor_type *type);
 
+/**
+ * @defgroup BT_MESH_SENSOR_FORMATS Sensor channel formats
+ * @brief All available sensor channel formats in the mesh device properties
+ * Specification.
+ * @{
+ */
+
+/* Percentage formats */
```

Comment:
One comment I'll post now anyway. If we make these types available in the API, they need to have a description. Single comment for a set of formats can't be parsed by Doxygen (unless I'm mistaken). Though these formats can be still grouped in Doxygen.

Reply:
Grouped by format I mean (`percentage formats`, environmental formats`, etc).

Reply:
This will be replaced by my commit. I already made these public and documented them.

Reply:
Right, sorry, I forgot this.